NOTICE

```diff
@@ -0,0 +1,7 @@
+Ensembl
+See the NOTICE file distributed with this work for additional information
```

Comment:
Does line 2 need to contain the copyright year information rather than this NOTICE text as in the [ensembl-io PR](https://github.com/Ensembl/ensembl-io/pull/170/files#diff-dfb14fbb9e7d095209ec4cfd621069437bf9c442ff9de9d4ce889781bd0fefcf)?

Reply:
Definitely yes! THANKS @nwillhoft